melos.yaml

```diff
@@ -27,6 +27,35 @@ scripts:
   test:unit:ios: >
     ./.circleci/test_all_plugins.sh ios-test && exit 0
 
+  test:integration:
+    run: melos run test:integration:android && melos run test:integration:ios
+    description: Run all integration tests for all package example apps on Android and iOS simulators. Skips if no tests available.
+      - Requires running Android and iOS simulators.
+    
+  test:integration:android:
+    run: melos exec "flutter drive --driver=test_driver/integration_test.dart --target=integration_test/main_test.dart -d sdk"
```

Comment:
If the `-d` flag is omitted, it prompts you to choose one of your available devices, same for if you have multiple devices matching the argument for `-d`. I'm not sure if there is a better way to specify that in a way that is most compatibly for most people. If there is matching device available then the process fails for that device and they get an error message that says that (similar to running `flutter run`).

Reply:
I guess I was wondering what the benefit is to having these two commands, but I suppose it's for automation!

Reply:
I have found while working with them I often just run the android tests to debug it because it runs faster than ios (or both) while trying something I expect to either work or not work universally. Then, I can run w both when it seems to work well on android.